test/common/http/dispatcher_test.cc

```diff
@@ -59,23 +59,23 @@ class DispatcherTest : public testing::Test {
   Http::ContextImpl http_context_;
   AsyncClientImpl client_;
   Dispatcher http_dispatcher_;
-  envoy_observer observer_;
+  envoy_http_callbacks callbacks_;
```

Comment:
this name collides with the MockAsyncClientCallbacks: https://github.com/lyft/envoy-mobile/pull/406/files#diff-04eab2469b3cc12377df7e5b0015914aR49